packages/terra-show-hide/src/ShowHideFocuser.jsx

```diff
@@ -0,0 +1,89 @@
+import React, { forwardRef, useState } from 'react';
+import PropTypes from 'prop-types';
+import classNamesBind from 'classnames/bind';
+import { v4 as uuidv4 } from 'uuid';
+import styles from './ShowHide.module.scss';
+
+const cx = classNamesBind.bind(styles);
+
+const propTypes = {
+  /**
+   * Text that precedes the focusable portion.
```

Comment:
If the prop is focusableText, is this comment correct.  The comment states that it is the text that precedes the focusable portion.

Reply:
Good catch, it was actually the other way around. Fixed in https://github.com/cerner/terra-core/pull/3838/commits/c8f17c023738231996cc1c6baa69f7c03d4a046b